pkg/istio-agent/agent.go

```diff
@@ -691,7 +691,10 @@ func socketHealthCheck(ctx context.Context, socketPath string) error {
 	if err != nil {
 		return err
 	}
-	conn.Close()
+	err = conn.Close()
+	if err != nil {
+		log.Infof("connection is not closed with error: %v", err)
```

Comment:
sorry to be nit-picky but since I already had another comment...

```suggestion
		log.Infof("connection is not closed: %v", err)
```

Is the go recommendation for error formats